composer.lock

```diff
@@ -9,16 +9,16 @@
     "packages-dev": [
         {
             "name": "composer/ca-bundle",
-            "version": "1.1.1",
```

Comment:
Please also update `composer.json`

Reply:
I guess you meant we should update the composer.json to the latest available versions of the packages we depend on.
I did that with https://github.com/Ocramius/PackageVersions/pull/93/commits/ece22f84e92e4b79de79b1a29517070c9297bdce

if you meant something different, just let me know. was a bit confued since composer/ca-bundle is only a transitive dependency which does not get mentioned in composer.json

Reply:
Yeap, thanks!